Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/polygon_soup_to_polygon_mesh.h

```diff
@@ -37,13 +37,30 @@ namespace CGAL {
 namespace Polygon_mesh_processing {
 namespace internal {
 
+template <typename PolygonMesh>
+struct Default_PS_to_PM_visitor
+{
+  typedef typename boost::graph_traits<PolygonMesh>::vertex_descriptor          vertex_descriptor;
+  typedef typename boost::graph_traits<PolygonMesh>::face_descriptor            face_descriptor;
+
+  template <typename PID>
+  void on_vertex_creation(const PID pid, const vertex_descriptor v, const PolygonMesh& pmesh) const
+  {
+    std::cout << "Point[" << pid << "] gives vertex " << v << std::endl;
+  }
+
+  template <typename FID>
+  void on_face_creation(const FID fid, const face_descriptor f, const PolygonMesh& pmesh) const
+  {
+    std::cout << "Face[" << fid << "] gives faces " << f << std::endl;
+  }
```

Comment:
It seems like you can hardcode `std::size_t` instead of `PID` and `FID`  template parameters.

And be careful that the default visitors cannot do anything. And be careful of warnings about unused parameters.

Reply:
The visitors are currently dumping messages just to illustrate the API for you. Once it is determined that this works for you, I'll clean them.